src/GuppyDataExplorer/configTypeDef.js

```diff
@@ -46,6 +46,7 @@ export const ButtonConfigType = PropTypes.shape({
   terraExportURL: PropTypes.string,
   terraTemplate: PropTypes.arrayOf(PropTypes.string),
   sevenBridgesExportURL: PropTypes.string,
+  registerUsersForDownload: PropTypes.bool,
```

Comment:
I would rename this to something like `loginForButtonFeature` which reflects the meaning of this option more correct

Because you are only redirecting user to login page by this option, and it is depending on Fence to decide wether to show the registration form or not

Reply:
also you should update the portal config markdown doc with this new options

Reply:
Updated config variable name and added the description of the config variable to portal_config.md file